docs/AI-for-security/connect-to-vertex.asciidoc

```diff
@@ -0,0 +1,119 @@
+[[connect-to-vertex]]
+= Connect to Google Vertex
+
+:frontmatter-description: Set up a Google Vertex LLM connector.
+:frontmatter-tags-products: [security]
+:frontmatter-tags-content-type: [guide]
+:frontmatter-tags-user-goals: [get-started]
+
+This page provides step-by-step instructions for setting up a Google Vertex AI connector for the first time. This connector type enables you to leverage Vertex AI's large language models (LLMs) within {elastic-sec}. You'll first need to enable Vertex AI, then generate an API key, and finally configure the connector in your {elastic-sec} project.
+
+IMPORTANT: Before continuing, you should have an active project in one of Google Vertex AI's https://cloud.google.com/vertex-ai/docs/general/locations#feature-availability[supported regions]. 
+
+[discrete]
+== Enable the Vertex AI API
+
+1. Log in to the GCP console and navigate to **Vertex AI → Vertex AI Studio → Overview**. 
+2. If you're new to Vertex AI, the **Get started with Vertex AI Studio** popup appears. Click **Vertex AI API**, then click **ENABLE**.
```

Comment:
Optional nit: I'm not a fan of replicating all-caps for UI labels, because it feels like yelling (especially if it happens more than just once or twice in a doc). But I don't think we have any style guidelines about it, so it might come down to personal preference.